src/commands/mod.rs

```diff
@@ -2783,55 +2806,66 @@ fn rebase_branch(
     command: &CommandHelper,
     workspace_command: &mut WorkspaceCommandHelper,
     new_parents: &[Commit],
-    branch_str: &str,
+    branch_commits: &IndexSet<Commit>,
```

Comment:
FYI, I'm thinking of replacing these with `impl IntoIterator<Item = &'a Commit> + Copy` as explained [here](https://stackoverflow.com/a/75503605/563359) here, in `rebase_descendants`, `rebase_commit`, etc.